tests/gis_tests/geoapp/test_functions.py

```diff
@@ -279,6 +279,11 @@ def test_area_lookups(self):
         with self.assertRaisesMessage(ValueError, 'AreaField only accepts Area measurement objects.'):
             qs.get(area__lt=500000)
 
+    @skipUnlessDBFeature("has_LineLocatePoint_function")
+    def test_line_locate_point(self):
+        pos_expr = functions.LineLocatePoint(LineString((0, 0), (0, 3), srid=4326), Point(0, 1, srid=4326))
+        self.assertAlmostEquals(State.objects.annotate(pos=pos_expr).first().pos, 0.3333333)
```

Comment:
I guess this is like `assertEqual` where the "Equals" version is a deprecated alias.